Loop/View Controllers/BolusViewController.swift

```diff
@@ -24,15 +24,23 @@ final class BolusViewController: ChartsTableViewController, IdentifiableClass, U
         case chart = 0
         case carbEntry
         case notice
+        case date
+        case model
         case recommended
         case entry
     }
 
+    private var maximumDateFutureInterval = TimeInterval(hours: 48)
```

Comment:
Can be a `let`; I also think we can shorten this interval. Maybe use the future interval for carb entry as a guide.

Reply:
Makes sense. This PR was motivated by a desire to allow for long-acting insulin to be logged (still a WIP), which is the reason for the long interval, but in this context the carb entry duration would be better.